Added ente

vkeerthivikram opened the pull request:
**Type of pull request:** product addition

**Related issues:** 
#185 

---

Added ente following guidlines mentioned in [https://privacyspy.org/contribute/](https://privacyspy.org/contribute/)

vkeerthivikram committed:
Added ente

doamatto commented on products/ente.toml:
```diff
@@ -0,0 +1,104 @@
+name = "ente"
```
Should be capitalised.

vkeerthivikram replied:
Done

doamatto commented on products/ente.toml:
```diff
@@ -0,0 +1,104 @@
+citations = [
+  "We do not sell your personal information, nor do we intend to do so."
+]
+notes = [
```
Note isn't necessary.

doamatto commented on products/ente.toml:
```diff
@@ -0,0 +1,104 @@
+[rubric.data-breaches]
+value = "yes-72"
+citations = ["In the event of a personal data breach, we will notify you within seventy-two (72) hours via email."]
+notes = [
```
Note isn't needed

doamatto commented on products/ente.toml:
```diff
@@ -0,0 +1,104 @@
+
+[rubric.data-collection-reasoning]
+value = "mostly"
+citations = ["We collect and store only the bare minimum amount of information necessary to fulfill our role as a service provider."]
```
See Section 4. The score and citations should be updated.

doamatto commented on products/ente.toml:
```diff
@@ -0,0 +1,104 @@
+[rubric.data-collection-reasoning]
+value = "mostly"
+citations = ["We collect and store only the bare minimum amount of information necessary to fulfill our role as a service provider."]
+notes = ["The privacy policy provides a clear list of what information is collected and generally explains why, but doesn't give specific reasoning for each data point collected. They emphasize minimal data collection overall."]
```
Note isn't needed

doamatto commented on products/ente.toml:
```diff
@@ -0,0 +1,104 @@
+
+[rubric.data-deletion]
+value = "yes-automated"
+citations = ["After account termination, we may retain your Data for sixty (60) days, or as warranted by your jurisdiction (\"Retention Period\"), unless an enforcement action is likely under ourTerms. If there is no enforcement action likely or commenced and Retention Period has expired, your Data that identifies you will be anonymized."]
```
See Section 9 and Section 17. Since only contact is mentioned as a way to delete information, we have to assume (even though it can be done yourself) that it can only be done via e-mail or post.

You can add the automated process as a note.

doamatto commented on products/ente.toml:
```diff
@@ -0,0 +1,104 @@
+[rubric.data-deletion]
+value = "yes-automated"
+citations = ["After account termination, we may retain your Data for sixty (60) days, or as warranted by your jurisdiction (\"Retention Period\"), unless an enforcement action is likely under ourTerms. If there is no enforcement action likely or commenced and Retention Period has expired, your Data that identifies you will be anonymized."]
+notes = ["The privacy policy states that users can terminate their account, which leads to the permanent deletion of personal data after a retention period. While there's a delay before full deletion, this still satisfies the criteria for permanent deletion."]
```
This note isn't needed

doamatto commented on products/ente.toml:
```diff
@@ -0,0 +1,104 @@
+[rubric.history]
+value = "last-modified"
+citations = ["Last Updated: Sep 25, 2024"]
+notes = ["The privacy policy includes a 'Last Updated' date at the beginning, which indicates when the policy was last modified. However, it does not provide a full history of changes or previous versions."]
```
Note isn't needed

doamatto commented on products/ente.toml:
```diff
@@ -0,0 +1,104 @@
+[rubric.law-enforcement]
+value = "strict"
+citations = ["We will disclose personal information (i) to comply with any court order, law, or legal process, including to respond to any government or regulatory request"]
+notes = ["The privacy policy states that Ente will only disclose personal information to law enforcement in response to legal processes like court orders or government requests, which aligns with the 'strict' criteria of 'Only when required by a court order or subpoena'.", "It's important to note that Ente emphasizes end-to-end encryption for user files and metadata, stating 'Your Files, their me ...
```
Note isn't needed

doamatto commented on products/ente.toml:
```diff
@@ -0,0 +1,104 @@
+
+[rubric.list-collected]
+value = "exhaustively"
+citations = ["Information you Provide: At the time of registration, or through your use of our Services, you will provide us with 1. Your email address; 2. Referral details including referrers and people you have referred; 3. Email addresses you choose to share your Files with; 4. Our Communications with you and records or copies of such communications; 5. Other personal information you provide t ...
```
Should also include Section 3d

doamatto commented on products/ente.toml:
```diff
@@ -0,0 +1,104 @@
+[rubric.list-collected]
+value = "exhaustively"
+citations = ["Information you Provide: At the time of registration, or through your use of our Services, you will provide us with 1. Your email address; 2. Referral details including referrers and people you have referred; 3. Email addresses you choose to share your Files with; 4. Our Communications with you and records or copies of such communications; 5. Other personal information you provide t ...
+notes = ["The privacy policy provides a detailed and specific list of the types of personal data collected, both provided by the user and automatically collected. This level of detail meets the criteria for 'exhaustively' listing the collected personal data."]
```
Note isn't needed

doamatto commented on products/ente.toml:
```diff
@@ -0,0 +1,104 @@
+
+[rubric.noncritical-purposes]
+value = "na"
+citations = ["We collect and store only the bare minimum amount of information necessary to fulfill our role as a service provider."]
```
Citation isn't relevant

doamatto commented on products/ente.toml:
```diff
@@ -0,0 +1,104 @@
+[rubric.noncritical-purposes]
+value = "na"
+citations = ["We collect and store only the bare minimum amount of information necessary to fulfill our role as a service provider."]
+notes = ["The privacy policy emphasizes minimal data collection and does not mention any non-critical data collection or usage. There are no opt-out or opt-in options mentioned for non-critical purposes, suggesting that no data is collected or used for such purposes.", "The policy explicitly states that they do not sell personal information or engage in behavioral marketing practices."]
```
Simpler to mention that there's no mention in Section 4 of noncritical usage

doamatto commented on products/ente.toml:
```diff
@@ -0,0 +1,104 @@
+[rubric.revision-notify]
+value = "no"
+citations = ["We will update this privacy policy as needed so that it is current, accurate, and as clear as possible. Your continued use of our Services confirms your acceptance of our updated Privacy Policy."]
+notes = [
```
Note isn't needed

doamatto commented on products/ente.toml:
```diff
@@ -0,0 +1,104 @@
+value = "yes"
+citations = [
+  "We have implemented measures designed to secure your personal information from accidental loss and from unauthorized access, use, alteration, and disclosure. Specifically (a) all information you provide to us is stored on our secure servers behind firewalls, (b) our website and app use an SSL certificate, receive regular security scans, penetration tests and regular malware scans; (c) we requi ...
+  "Your Files, their metadata and any derived indexes (including but not limited to Biometric Information such as face geometry) are stored end-to-end encrypted."
```
Citation isn't very relevant

doamatto commented on products/ente.toml:
```diff
@@ -0,0 +1,104 @@
+  "We have implemented measures designed to secure your personal information from accidental loss and from unauthorized access, use, alteration, and disclosure. Specifically (a) all information you provide to us is stored on our secure servers behind firewalls, (b) our website and app use an SSL certificate, receive regular security scans, penetration tests and regular malware scans; (c) we requi ...
+  "Your Files, their metadata and any derived indexes (including but not limited to Biometric Information such as face geometry) are stored end-to-end encrypted."
+]
+notes = [
```
Current note isn't needed. Note should mention audit : https://ente.io/blog/cryptography-audit/. Given its age, I wouldn't personally say the score should be changed to `yes-independent-audits`.

doamatto commented on products/ente.toml:
```diff
@@ -0,0 +1,104 @@
+]
+
+[rubric.security]
+value = "yes"
```
Should be `somewhat`.

vkeerthivikram replied:
Changed it to somewhat and mentioned on how the deletion occurs (using the account settings) in the notes

doamatto commented on products/ente.toml:
```diff
@@ -0,0 +1,104 @@
+
+[rubric.third-party-access]
+value = "yes-specified-critical"
+citations = ["We do not sell, trade, rent, or otherwise transfer personal information to others, unless we provide you with advance notice. There are times when Personal Information that you have shared with us may be shared by Ente with others to enable us to provide you over Services, including contractors, service providers, and third parties (\"Partners\") and subsidiaries.", "Your Files, the ...
```
Second citation isn't relevant.

Third citation shouldn't have brackets and "list of providers including"

doamatto commented on products/ente.toml:
```diff
@@ -0,0 +1,104 @@
+[rubric.third-party-access]
+value = "yes-specified-critical"
+citations = ["We do not sell, trade, rent, or otherwise transfer personal information to others, unless we provide you with advance notice. There are times when Personal Information that you have shared with us may be shared by Ente with others to enable us to provide you over Services, including contractors, service providers, and third parties (\"Partners\") and subsidiaries.", "Your Files, the ...
+notes = [
```
Note isn't necessary

doamatto commented on products/ente.toml:
```diff
@@ -0,0 +1,104 @@
+value = "critical-only"
+citations = [
+  "We collect payment invoices provided to us by our third-party payment processors, which includes details of your Subscription Plan and any payments made by you in favor of Ente in order to receive Services from us. We do not collect or store any credit cards or bank information.",
+  "Ente uses the following third-party service providers for the provision of services as detailed under the Terms, as applicable: [list of providers including Apple, Google, Stripe, BitPay, PayPal, Scaleway, Backblaze, Cloudflare, Amazon, Hetzner, FeatureMonkey, Simple Analytics, Zoho, Grafana, Open Street Maps]"
```
Citation isn't necessary

doamatto commented on products/ente.toml:
```diff
@@ -0,0 +1,104 @@
+  "We collect payment invoices provided to us by our third-party payment processors, which includes details of your Subscription Plan and any payments made by you in favor of Ente in order to receive Services from us. We do not collect or store any credit cards or bank information.",
+  "Ente uses the following third-party service providers for the provision of services as detailed under the Terms, as applicable: [list of providers including Apple, Google, Stripe, BitPay, PayPal, Scaleway, Backblaze, Cloudflare, Amazon, Hetzner, FeatureMonkey, Simple Analytics, Zoho, Grafana, Open Street Maps]"
+]
+notes = [
```
Note isn't necessary

doamatto commented on products/ente.toml:
```diff
@@ -0,0 +1,104 @@
+  "The policy emphasizes minimal data collection, suggesting that any third-party data collection is likely limited to critical purposes only.",
+  "Although not explicitly stated, services like payment processing or identity verification (if implemented) would fall under critical data collection from third parties.",
+  "The 'critical-only' value is appropriate given the focus on essential service provision and the absence of any indication of non-critical third-party data collection."
+]
```
Missing EOF it seems. Leave a blank line at the end of the file

doamatto reviewed (changes requested):
Few changes to make. Could we also find a better icon ? It's very squished.

doamatto commented:
On a somewhat related note, it looks like a lot of the notes were written by a chatbot or something of the like ? Politiwatch (afaik) doesn't bar contributions of that kind, but it would be interesting to know if that was the case. Apologies for the presumption if that's not the case

vkeerthivikram committed:
Make Changes requested in the pull request

vkeerthivikram commented:
> On a somewhat related note, it looks like a lot of the notes were written by a chatbot or something of the like ? Politiwatch (afaik) doesn't bar contributions of that kind, but it would be interesting to know if that was the case. Apologies for the presumption if that's not the case

Well, I did rewrite whatever notes I wrote, using an AI to make it look a bit professional. Apologize if that was an issue, I will put my original notes out there instead.

vkeerthivikram committed:
add new icon for ente

vkeerthivikram commented:
> Few changes to make. Could we also find a better icon ? It's very squished.

Regarding the icon, in the ente press kit, the icon is rectangular shaped, but it fails the tests, so I resized it to square shaped. Is it ok if I submit the rectangular shaped icon?

Edit: Found another more minimalistic icon from their github and added it

doamatto committed:
fix(product): changes to Ente

- Reduce potential bias in production description
- Reduced some blankspaces
- Added missing citation to `rubric.third-party-collection`
- Added missing citation to `rubric.security`
- Reworded note for `rubric.security`

doamatto commented:
Looks all good now. Thanks ! ^_^
